### ISSUE TITLE
Branch stats are inconsistent

### ISSUE THREAD

**Issue by MeBrei** [radicle-dev/radicle-upstream#1376]
<img width="1345" alt="Screenshot 2020-11-26 at 11 23 31" src="https://user-images.githubusercontent.com/16262137/100339261-ef0dcd80-2fd9-11eb-8c00-18f25f3506d2.png">

After forking a project, the branch stats are not aligning with the maintainers stats. 
Do we always show our "own" stats or the ones from the maintainers/creator of the project?

**Comment by FintanH** [comment]
Another option would be to show the selected peers stats

**Comment by cloudhead** [comment]
It should be the stats of the selected `(remote, branch)` tuple

**Comment by xla** [comment]
> EDIT: And obviously if we're talking branch count, then just the `remote`, as finto suggested

Just for devils advocacy, it might be confusing as the numbers are shown in the context of the project meta and not close to the peer selected, just food for thouht. Generally I agree with that sentiment.

**Comment by cloudhead** [comment]
Yeah, was thinking the same. Perhaps the contributor count should be some kind of total? Looks like Github does this since they updated their UI

**Comment by MeBrei** [comment]
So we have all places covered: In the list of "my" projects, the branch stats should show my number of branches on the project? (I think that is how it is right now)

**Comment by cloudhead** [comment]
It should use the stats of the "default fork", ie. the same that is shown when you click on the project.

**Comment by cloudhead** [comment]
I believe that's your own for the Projects tab, and the maintainer's in the Follow tab.

**Comment by xla** [comment]
> I believe that's your own for the Projects tab, and the maintainer's in the Follow tab.

The latter part might be not working as expected.

**Comment by cloudhead** [comment]
cf. https://github.com/radicle-dev/radicle-upstream/issues/1243
